scripts/policy/frameworks/analyzer/deprecated-dpd-log.zeek

```diff
@@ -0,0 +1,89 @@
+##! Creates the now deprecated dpd.logfile.
+
```

Comment:
Could make sense to add a `# Remove in v8.1` comment here so that it doesn't fall through the cracks when Tim greps for deprecations after 8.0?